HW_7/view.py

```diff
@@ -0,0 +1,110 @@
+from logger import log
+
+
+@log
+def greatings():
+    '''Вывод приветствия.'''
+
+    print('Справочник')
+    pass
+
+
+@log
+def menu() -> int:
+    """
+    Вывод меню, запрос данных от пользователя.
+    :return:
+    0 - Выход
+    1 - Загрузить из файла и вывести на экран
+    2 - Добавить новую запись
+    3 - Редактировать запись по id
+    4 - Поиск по фамилии
+    """
+    print('Меню')
+    print(
+        '0 - Выход \n1 - Загрузить из файла и вывести на экран \n2 - Добавить новую запись \n3 - Редактировать запись по id \n4 - Поиск по фамилии \n')
+    return int(input("Введите пункт меню: "))
```

Comment:
Можно все поместить внутрь функции input
```suggestion
    return int(input(
        'Меню\n'
        '0 - Выход\n'
        '1 - Загрузить из файла и вывести на экран\n'
        '2 - Добавить новую запись\n'
        '3 - Редактировать запись по id\n'
        'Введите пункт меню: '
    ))
```